languages/ruby/exercises/concept/booleans/.docs/instructions.md

```diff
@@ -11,7 +11,7 @@ Attendee.new(100).has_pass?
 
 ## 2. Check if an attendee fits a ride
 
-Implement the `fits_ride?` method to see if an attendee fits a ride based on their height and the minimum height of required by the ride.
+Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height and the minimum height of required by the ride.
```

Comment:
Also, this line doesn't give the implementation detail of whether to use a Boolean or a boolean… so `pass?`'s implementation message, at least, seems out of style.

Reply:
Of course, for these comments, these changes don't quite fit the purpose of the pull request, so of course, this would also be a non-blocking observation.